### ISSUE TITLE
JSON data for query generation

### ISSUE THREAD

**Issue by jiahao-mobi** [OSU-NLP-Group/TravelPlanner#21]
Hi, I am wondering whether the JSON input data that are used in generating the query (in `build_query_generation_prompt` function) will be released?

**Comment by hsaest** [comment]
Hi,

Thank you for your interest in our work. 

Actually, we have released the JSON format data of the training set and validation set. However, in order to avoid possible cheating, we keep the JSON format data of the test set for evaluation.

**Comment by hsaest** [comment]
HI,

Sorry to say that we avoid any form of test set release. Our query construction process is easy to follow and the scripts are provided, so in order to have similar data for use in the industry field, we encourage you to re-create some new queries.

**Comment by jiahao-mobi** [comment]
Hi, thank you for the response. Fully understand your concern!
